DigitalLearningSolutions.Web/Views/Register/Password.cshtml

```diff
@@ -1,4 +1,4 @@
-﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@using DigitalLearningSolutions.Web.ViewModels.Common

```

Comment:
Not sure what happened here. Byte Order Mark? It's not a line ending change anyway.

Reply:
Yeah, looks like the byte order mark's been removed.